### PR TITLE
Updated remove fake to v3

### PR DESCRIPTION
**Pull request by Akash-Mair**

### CODE REVIEW

File: docs/recipes/build/remove-fake.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Delete `Build.fs`, `Build.fsproj`, `Helpers.fs` at the root of the solution.
+#### 2. Dependencies
+Find the following dependencies inside the `paket.dependencies` file that’s also at the root of the solution and delete them.
+```fsharp
+nuget Fake.Core.Target
+nuget Fake.IO.FileSystem
+nuget Farmer
+```
+
+Also remove the `paket.references` at the root of the solution which should contain:
```
Comment (isaacabraham) [inline docs/recipes/build/remove-fake.md:18]: Remove this section and just add the file to the list of files on line 9.

File: docs/recipes/build/remove-fake.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+[FAKE](https://fake.build/) is a tool for build automation. The standard SAFE template comes with a [ready-made build project](/template-safe-commands) at the root of the solution that provides support for many common SAFE tasks.
+
+If you would prefer not to use FAKE, you can of course simply ignore it, but this recipes shows how to completely remove it from your repository. It is important to note that having removed FAKE, you will have to follow a more manual approach to each of these processes. This recipe will only include instructions on how to build and deploy the application after removing FAKE.
+
+> Note that the minimal template does not use FAKE by default, and **this recipe only applies to the standard template**.
+
+#### 1. Build project
+Delete `Build.fs`, `Build.fsproj`, `Helpers.fs` at the root of the solution.
+#### 2. Dependencies
+Find the following dependencies inside the `paket.dependencies` file that’s also at the root of the solution and delete them.
```
Comment (isaacabraham) [inline docs/recipes/build/remove-fake.md:11]: Replace opening the file with `dotnet paket remove Fake.Core.Target` etc.

File: docs/recipes/build/remove-fake.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+Also remove the `paket.references` at the root of the solution which should contain:
+```
+Fake.Core.Target
+Fake.IO.FileSystem
+Farmer
+```
+
+#### 3. Paket Install
```
Comment (isaacabraham) [inline docs/recipes/build/remove-fake.md:25]: This step can be removed if you use Paket remove (as above).

File: docs/recipes/build/remove-fake.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+Then, execute `paket install` in your terminal at the root of the solution. This will remove the dependencies that the Build project was using.
+
+## Running the App
```
Comment (isaacabraham) [inline docs/recipes/build/remove-fake.md:28]: Is this section not shown for how to run the app generated by the minimal template option? If so, don't duplicate here, just link to that.

**Review by isaacabraham — CHANGES_REQUESTED** [review]

File: docs/recipes/build/remove-fake.md (hunk 4 above)
Comment (Akash-Mair) [reply docs/recipes/build/remove-fake.md:19]: I don't think there is a section showing how to run the minimal app but I do think it's in a few places. Would it be better to put this in a new section and reference that in all the places we show how to run the minimal app?

Comment (isaacabraham) [reply docs/recipes/build/remove-fake.md:19]: Yes, definitely.

Comment (isaacabraham) [reply docs/recipes/build/remove-fake.md:19]: But that should be done as a separate PR I think.

File: docs/recipes/build/remove-fake.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+Remove the following dependencies 
+```fsharp
+dotnet paket remove Fake.Core.Target
+dotnet paket remove Fake.IO.FileSystem
+dotnet paket remove Farmer
+```
+
+## Running the App
+Now that you have the FAKE dependencies removed, you will have to separately run the server and the client.
+#### 1. Start the Client
```
Comment (isaacabraham) [inline docs/recipes/build/remove-fake.md:21]: Might it be better to start the server first and then the client? That way an "on initialisation" calls will work.

File: docs/recipes/build/remove-fake.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+[FAKE](https://fake.build/) is a tool for build automation. The standard SAFE template comes with a [ready-made build project](/template-safe-commands) at the root of the solution that provides support for many common SAFE tasks.
+
+If you would prefer not to use FAKE, you can of course simply ignore it, but this recipes shows how to completely remove it from your repository. It is important to note that having removed FAKE, you will have to follow a more manual approach to each of these processes. This recipe will only include instructions on how to build and deploy the application after removing FAKE.
+
+> Note that the minimal template does not use FAKE by default, and **this recipe only applies to the standard template**.
+
+#### 1. Build project
+Delete `Build.fs`, `Build.fsproj`, `Helpers.fs`, `paket.references` at the root of the solution.
+#### 2. Dependencies
+Find the following dependencies inside the `paket.dependencies` file that’s also at the root of the solution and delete them.
```
Comment (isaacabraham) [inline docs/recipes/build/remove-fake.md:11]: This line should be removed as well?

File: docs/recipes/build/remove-fake.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+The app will now be running at `http://0.0.0.0:8080/`. Navigate to this address in a browser to see your app running.
+
+## Bundling the App
+See [this guide](../build/bundle-app.md#2-im-using-the-minimal-template) to learn how to package a SAFE application for deployment to e.g. Azure.
+
+---
+
+You can now find the outcome of the bundling process in the `deploy` folder at the root of the solution.
```
Comment (isaacabraham) [inline docs/recipes/build/remove-fake.md:45]: This line can go.

**Review by isaacabraham — CHANGES_REQUESTED** [review]

**Review by isaacabraham — APPROVED** [review]

**Pull request merged**